test/triples.csv

```diff
@@ -36,3 +36,6 @@ test_var_3,typeOf,StatisticalVariable,
 test_var_3,measuredProperty,test_var_3,
 test_var_3,statType,measuredValue,
 test_var_3,name,,"non-place entity variable"
+ein/1,typeOf,Company,
```

Comment:
How are these files used? Are there tests that use them?

Reply:
Yes, these csvs are loaded into datacommons.db for testing via "./import.sh". I will document them too.